### PR TITLE
{phys}[foss/2022a] Block v1.5.3-20200525

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

Depends on:
* #16714

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/Block/Block-1.5.3-20210723-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+easyblock = 'MakeCp'
+
+name = 'Block'
+version = '1.5.3-20210723'
```
Comment (branfosj) [inline easybuild/easyconfigs/b/Block/Block-1.5.3-20210723-foss-2022a.eb:4]: I think this source code is 1.1.1.

From `block.spin_adapted-serial --version`
```
Block 1.1.1
```

Whereas the 1.5.3 downloaded from https://github.com/pyscf/Block/tree/master/bin reports back as `1.5`.

I cannot get the 1.5.3 source code link from https://sanshar.github.io/Block/build.html to work to check in more detail.

**Review by branfosj — DISMISSED** [review]

File: easybuild/easyconfigs/b/Block/Block-1.5.3-20210723-foss-2022a.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/b/Block/Block-1.5.3-20210723-foss-2022a.eb:4]: Good question, this easyconfig is already using the sources from https://github.com/pyscf/Block, which is the reference repo according to https://github.com/sanshar/Block/issues/59

Maybe the binaries were updated to v1.5.3 but the source code is still old. I asked the question upstream, see
https://github.com/sanshar/Block/issues/59#issuecomment-1342770661

**Comment by bogdanoff** [comment]
Actually it happened that [StackBlock](https://github.com/sanshar/StackBlock) was referred to as Block 1.5. The released Block 1.5.3  tarball  is a slice of the StackBlock repository. At some point I extracted all the releases to https://github.com/bogdanoff/Block/tree/release-1.5 . I believe `Block 1.5` was never printed by the code, even PySCF determines the version of Block indirectly: https://github.com/pyscf/dmrgscf/blob/master/pyscf/dmrgscf/dmrgci.py#L1002

**Comment by lexming** [comment]
@bogdanoff thanks a lot for the info, that clarifies the issue with the version of the source code. I'll update this PR to use the correct sources.

**Comment by boegel** [comment]
@lexming Any updates?

File: easybuild/easyconfigs/b/Block/Block-1.5.3-20210723-foss-2022a.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/b/Block/Block-1.5.3-20210723-foss-2022a.eb]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16711/commits/d14524f868b0a6bb0adc699bdff956aadafca829

**Comment by lexming** [comment]
@boegel fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16711/commits/d14524f868b0a6bb0adc699bdff956aadafca829

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3129.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/1555f5ae31a2d7ed786b695837bc67e1 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @lexming!

**Pull request merged**